prototype2/sonde/ideas/DataTest.cpp

```diff
@@ -7,9 +7,11 @@
 #include <sonde/ideas/TestData.h>
 #include <test/TestBase.h>
 
+using namespace Sonde;
+
 class IDEASDataTest : public TestBase {
 protected:
-  SoNDeGeometry geometry;
+  Geometry geometry;
   IDEASData *readout;
   virtual void SetUp() {
     readout = new IDEASData(&geometry);
```

Comment:
This and the next line should really be replaced with the use of `std::unique_ptr`.